Monika After Story/game/script-topics.rpy

```diff
@@ -3587,6 +3587,35 @@ label monika_chloroform:
     m 1tsb "You're so cute when you're nervous."
     return
 
+init 5 python:
+    if os.path.isfile(os.path.expandvars("%APPDATA%") + '\RenPy\JustYuri\persistent'):
```

Comment:
Addition of events shouldn't be done in an if statement within the python block. Rather the event label should be given the `conditional` property instead.

Reply:
init 5 python:
  addEvent(Event(persistent.event_database,eventlabel="just_yuri",category=['club members'],prompt="Just Yuri mod",random=True))

label just_yuri:
    if os.path.isfile(os.path.expandvars("%APPDATA%") + '\RenPy\JustYuri\persistent'):
       m 1esc "[player]..."
       m 4esc "A thing drew to my attention."
       m 4esc "Did you..."

Is like that?

Reply:
A little more like:
```init 5 python:
   addEvent(
       Event(
            eventlabel="just_yuri",
            category=['club members'],
            prompt="Just Yuri mod",
            conditional=(
                "condition"
            )
       )
   )
```